### PR TITLE
Tizen devices fixes

### PR DESCRIPTION
**Pull request by justinasRm**
## Description

Running `npx rnv run -p tizen`, now you can select from real connected devices.
Running `npx rnv target list -p tizen` displays the real device IP only(no unnecessary, default info)

## Npm releases

n/a

### CODE REVIEW

**Comment by pauliusguzas** [comment]
@justinasRm could you write down impacted commands, what is expected of them?

**Review by pauliusguzas — CHANGES_REQUESTED** [review]
`npx rnv run -p tizen -t`
`npx rnv run -p tizen` (when no default target added to renative.json)
shows this question (expected: choose from all targets -  sims and devices)
```
? Could not find or connect to the specified target (undefined). Would you like
to start an emulator?
```
Also seems like `npx rnv target launch -p tizen` tries to install app? for me TV get's stuck on "RNVanillaTV Loading" spinner, no controls on tv work. I am not sure what this command should to with devices, but definetely not install app

**Review by pauliusguzas — CHANGES_REQUESTED** [review]
```
? What emulator would you like to launch?
```
as this asks from emulators and devices, this question should be changed

also github actions task is failing

**Review by pauliusguzas — APPROVED** [review]

File: packages/sdk-tizen/src/deviceManager.ts [hunk 1]
```diff
@@ -71,44 +72,43 @@ const formatXMLObject = (
     return {};
 };
 
-export const launchTizenEmulator = async (name: string | true): Promise<boolean> => {
+export const launchTizenEmulator = async (name: string | true, hideDevices?: boolean): Promise<boolean> => {
```
Comment (Marius456) [inline packages/sdk-tizen/src/deviceManager.ts:75]: This method should only run/show emulators. Does it need to have that new hideDevices prompt?

Comment (justinasRm) [reply packages/sdk-tizen/src/deviceManager.ts:75]: Not sure why its only for emulators? The name can be changed.

**Review by Marius456 — APPROVED** [review]

**Pull request merged**